docker/monitoring/grafana/dashboards/datahub_dashboard.json

```diff
@@ -1485,16 +1837,31 @@
       },
       "id": 4,
       "panels": [],
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus"
```

Comment:
same

Reply:
these are created automatically , when we create we dashboard with uuid .
We have manually mapped datasource to variable for datasource in each panel and those changes are only reflecting in  git difference .